tests/libc/enc/helpers.c

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include "helpers.h"
+
+#include <dirent.h>
+#include <errno.h>
+#include <libgen.h>
+#include <openenclave/enclave.h>
+#include <openenclave/internal/tests.h>
+#include <search.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mount.h>
+#include <time.h>
+#include "mtest.h"
+
+extern char** __environ;
+
+#pragma STDC FENV_ACCESS ON
+
+#if defined(__x86_64__) || defined(_M_X64)
+#define XMM_OK
+#endif
+
+#if defined(XMM_OK)
+/* Type of the control word.  */
+typedef unsigned int fpu_control_t __attribute__((__mode__(__HI__)));
+/* Macros for accessing the hardware control word.  */
+#define _FPU_GETCW(cw) __asm__ __volatile__("fnstcw %0" : "=m"(*&cw))
+#define _FPU_SETCW(cw) __asm__ __volatile__("fldcw %0" : : "m"(*&cw))
+#endif
+
+// Space for global libc environment
+static char** environ = NULL;
+int t_status = 0;
+
+int device_init()
+{
+    OE_TEST(oe_load_module_host_file_system() == OE_OK);
+    OE_TEST(oe_load_module_host_socket_interface() == OE_OK);
+    // OE_TEST(oe_load_module_host_epoll() == OE_OK);
```

Comment:
Should we just remove this commented check (or do we know why we need to keep that)?